addok/helpers/collectors.py

```diff
@@ -102,3 +105,56 @@ def sorter(t):
             helper.add_to_bucket(keys)
             if helper.bucket_overflow:
                 break
+
+
+def extend_results_extrapoling_relations(helper):
```

Comment:
If we are not documenting this within the high-level documentation, maybe we should at least add detailed docstring?